packages/extension-collaboration/src/helpers/mapPositionFromTransaction.ts

```diff
@@ -0,0 +1,35 @@
+import type { MapPositionFromTransactionOptions, MapPositionFromTransactionResult } from '@tiptap/core'
+import { mapPositionFromTransaction as coreMapPositionFromTransaction } from '@tiptap/core'
+
+import { isChangeOrigin } from './isChangeOrigin.js'
+import { getYAbsolutePosition, getYRelativePosition } from './yRelativePosition.js'
+
+/**
+ * Returns the new position after applying a transaction. Handles both Y.js
+ * transactions and regular transactions.
+ */
+export function mapPositionFromTransaction(
```

Comment:
It's weird that the function itself is defined on the collaboration extension but the typing strictly named after this function lives in core.